backend/generator_configuration/json_parser.py

```diff
@@ -0,0 +1,54 @@
+from map import Map
+from sector import Sector
+import json
+
+
+def parse_map(json_filename):
+    with open(json_filename) as file:
+        # return json.load(file)
+        json_map = json.load(file)
+        sectors = []
+        for sector in json_map['sectors']:
+            sectors.append(Sector(sector_name=sector['sector_name'],
+                                  position=sector['position'],
+                                  time=sector['time'],
+                                  level_of_threat=sector['level_of_threat'],
+                                  temperature=sector['temperature'],
+                                  wind_direction=sector['wind_direction'],
+                                  wind_speed=sector['wind_speed'],
+                                  air_humidity=sector['air_humidity'],
+                                  litter_moisture=sector['litter_moisture'],
+                                  aerosol_concentration=sector['aerosol_concentration'],
+                                  carbon_dioxide_concentration=sector['carbon_dioxide_concentration']))
+        # print(sectors)
+        return Map(forest_name=json_map['forest_name'],
+                   height=json_map['height'],
+                   width=json_map['width'],
+                   sector_size=json_map['sector_size'],
+                   sectors=sectors)
+
+
+def parse_sector(json_filename, sector_name: str):
+    with open(json_filename) as file:
+        sectors = json.load(file)['sectors']
+        for sector in sectors:
+            if sector['sector_name'] == sector_name:
+                return Sector(sector_name=sector['sector_name'],
+                              position=sector['position'],
+                              time=sector['time'],
+                              level_of_threat=sector['level_of_threat'],
+                              temperature=sector['temperature'],
+                              wind_direction=sector['wind_direction'],
+                              wind_speed=sector['wind_speed'],
+                              air_humidity=sector['air_humidity'],
+                              litter_moisture=sector['litter_moisture'],
```

Comment:
Zmieniłbym "litter" na jedno z tych:
Litterfall, plant litter, leaf litter, tree litter, soil litter, or duff
To są angielskie nazwy ściółki wg wikipedii i wszystkie są bardziej zrozumiałe niż po prostu litter (który może naprowadzać na śmieci leżące w lesie, bo dokładnie tak samo się je nazywa)
(samo litter też jest poprawne, ale warto dodać ten dodatkowy wyraz dla rozjaśnienia)

Reply:
zgadzam się, zdecydowanie bardziej czytelne, zmieniłem na plant_litter_moisture